similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/AbstractPatientSimilarityView.java

```diff
@@ -130,6 +130,27 @@ public DocumentReference getReporter()
     }
 
     @Override
+    public String getOwnerName()
+    {
+        // TODO: Update once there is a convenient method for accessing the owner of the patient record.
```

Comment:
I doubt there will be a more convenient method of accessing the owner, since having an owner is and will be an optional feature. Many PhenoTips instances don't have owners and access rights at all, and I think we'll even switch the default that way, since it's easier to install the access rights module when needed than to remove it.